eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/EventHubSession.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs.implementation;
+
+import com.azure.core.amqp.AmqpLink;
+import com.azure.core.amqp.AmqpSession;
+import com.azure.core.amqp.Retry;
+import com.azure.eventhubs.EventReceiver;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+
+/**
+ * Represents an AMQP session that supports vendor specific properties and capabilities. For example, creating epoch
```

Comment:
nit:  Do we want to move away from the term `epoch` or are we using it as we get down the metal and direct service communication?

Reply:
Good catch. I'll revise the wording. :)